FirebaseInstallations/Source/Library/InstallationsStore/FIRInstallationsStore.h

```diff
@@ -23,17 +22,44 @@
 
 NS_ASSUME_NONNULL_BEGIN
 
+/// The user default suite name used to store data.
 extern NSString *const kFIRInstallationsStoreUserDefaultsID;
 
+/// The class is responsible for storing and accessing the installstions data.
 @interface FIRInstallationsStore : NSObject
 
+/**
+ *  @param storage The secure storage to save installations data.
+ *  @param accessGroup The Keychain Access Group to store and request the installstions data.
+ */
 - (instancetype)initWithSecureStorage:(FIRSecureStorage *)storage
                           accessGroup:(nullable NSString *)accessGroup;
 
 // TODO: Consider combining appID and appName to something like FIRInstallationsAppID
+/**
+ *  @param appID The Firebase(Google) Application ID.
+ *  @param appName The Firebase Application Name.
+ *
+ *  @return Returns a `FBLPromise` instance. The promise is resolved with a FIRInstallationsItem
+ * instance if there is a valid installation stored for `appID` and `appName`. The promise is
```

Comment:
Nit: indentation. I won't comment anymore on it but please make sure there are two spaces after each `*` for these types of documentation comments.

Reply:
./scripts/style.sh formats such comments with a single space after `*`, so I'll update my files to match it.